src/adapters/TelnetAdapter.ts

```diff
@@ -54,6 +64,7 @@ export class TelnetAdapter {
      * @param handler
      */
     public on(eventName: 'cannot-continue', handler: () => void);
+    public on(eventname: 'chanperf-event', handler: (output: ChanperfHistory) => void);
```

Comment:
We don't need the word "event" in this....it's already an event name. So the event can just be `chanperf`.